sktime/forecasting/base/_base.py

```diff
@@ -2472,4 +2472,9 @@ def _format_moving_cutoff_predictions(y_preds, cutoffs):
         cutoffs = [cutoff[0] for cutoff in cutoffs]
         y_pred = pd.concat(y_preds, axis=1, keys=cutoffs)
 
+    if not y_pred.index.is_monotonic_increasing:
+        y_pred = y_pred.sort_index()
+    if hasattr(y_pred, "columns") and not y_pred.columns.is_monotonic_increasing:
+        y_pred = y_pred.sort_index(axis=1)
```

Comment:
I think `y_pred` will have `columns` attribute if either it's a multivariate case or if we are doing interval/quantile prediction (or both). In these cases, as a user my expectation will be to have same order of variables as in my input dataset (incase I refer to those variables by position as opposed to name) and same order of quantiles/alpha specification.

From this point of view, I'm just a bit concerned about these two lines. We can do this in testing though.

Reply:
yes, you are right - I removed it again.

I think `update_predict` has a numbe of general problems:

* it actually fails in the combination case of multivariate and non-unique index
* the output in the non-unique case is clunky

I think it needs to be rewritten anyway seriously, an old attempt is here: https://github.com/sktime/sktime/pull/3447

A key question is what its docstring specification should even be in various cases of multivariate, hierarchical.

Should we open an issue?